src/main.ts

```diff
@@ -1,324 +1,1212 @@
+// npm install obsidian
+// npm install color-convert
+//
+// To run:
+//      npm install -g typescript
+//      npm install --save-dev @types/codemirror @types/dom-to-image @types/estree @types/node @types/randomcolor @types/resolve @types/tern
+//      npm run build
+
+
 import {
-	compile as compileTemplate,
-	TemplateDelegate as Template
+    compile as compileTemplate,
+    TemplateDelegate as Template
 } from 'handlebars';
 import { Editor, FileSystemAdapter, loadPdfJs, MarkdownView, Plugin, TFile, Vault, Notice } from 'obsidian';
+import convert from 'color-convert';    // For color conversion
+
+// Local modules
 import { loadPDFFile } from 'src/extractHighlight';
+import { saveDataToFile } from 'src/saveToFile';
 import { ANNOTS_TREATED_AS_HIGHLIGHTS, PDFAnnotationPluginSetting, PDFAnnotationPluginSettingTab } from 'src/settings';
 import { IIndexable, PDFFile } from 'src/types';
 
+
+
+// Formatting
+const title_lvl1 = "##### ";
+const lvl2_prefix = "- ";
+const lvl3_prefix = "\t- ";
+const sumr_prefix = "- ";
+const impt_prefix = "- ";
```

Comment:
```suggestion
const titleLvl1 = "##### ";
const lvl2Prefix = "- ";
const lvl3Prefix = "\t- ";
const sumrPrefix = "- ";
const imptPrefix = "- ";
```

Please use camelCase